package.json

```diff
@@ -121,52 +121,52 @@
     "@babel/plugin-syntax-import-meta": "7.10.4",
     "@babel/preset-env": "7.20.2",
     "@babel/preset-react": "7.18.6",
-    "@babel/preset-typescript": "7.18.6",
+    "@babel/preset-typescript": "7.21.0",
     "@electron/notarize": "1.2.3",
     "@lavamoat/allow-scripts": "2.3.0",
     "@lavamoat/preinstall-always-fail": "1.0.0",
-    "@ledgerhq/hw-transport-mocker": "6.27.10",
+    "@ledgerhq/hw-transport-mocker": "6.27.12",
     "@parcel/config-default": "2.8.3",
     "@parcel/core": "2.8.3",
     "@parcel/transformer-css": "2.8.3",
     "@parcel/transformer-stylus": "2.8.3",
     "@parcel/transformer-typescript-tsc": "2.8.3",
-    "@sentry/types": "7.33.0",
-    "@testing-library/dom": "8.20.0",
-    "@testing-library/react": "13.4.0",
+    "@sentry/types": "7.39.0",
+    "@testing-library/dom": "9.0.0",
+    "@testing-library/react": "14.0.0",
     "@testing-library/user-event": "14.4.3",
     "@types/cookie": "0.5.1",
     "@types/get-pixels": "3.3.2",
     "@types/hdkey": "2.0.1",
-    "@types/jest": "29.2.6",
-    "@types/node": "18.11.18",
+    "@types/jest": "29.4.0",
+    "@types/node": "18.14.2",
     "@types/node-fetch": "2.6.2",
     "@types/node-hid": "1.3.1",
     "@types/tar-fs": "2.0.1",
-    "@types/uuid": "9.0.0",
+    "@types/uuid": "9.0.1",
     "@types/ws": "8.5.4",
-    "electron": "22.0.3",
+    "electron": "23.1.1",
```

Comment:
what are the implications of this major version upgrade? seems like changing versions of node at a minimum would be quite impactful and require a fair bit of testing

Reply:
Absolutely, I think we should keep up with Electron versions and keep this in though.

Reply:
Here's the N18 release post:

https://nodejs.org/en/blog/announcements/v18-release-announce/